infra/app/env-template/main.tf

```diff
@@ -331,6 +341,19 @@ resource "aws_s3_bucket_cors_configuration" "doc_upload_cors" {
   }
 }
 
+module "refresh_s3_presigned_urls" {
+  source                       = "../../modules/task-scheduled"
+  schedule_name                = local.refresh_s3_presigned_urls_schedule_name
+  cluster_name                 = local.cluster_name
+  task_definition_family       = local.participant_service_name
+  container_task_override      = "{\"containerOverrides\": [{\"name\": \"${local.participant_service_name}\", \"command\": [\"npm\", \"run\", \"refresh-s3-urls\"]}]}"
```

Comment:
I'd love it if this were defined as a local JSON object and passed in but 🤷🏻‍♀️

Reply:
I tried that a few different ways and tf + ecs hated it 😞